server.js

```diff
@@ -1,25 +1,19 @@
-app.put('/slambook/:id', async (req, res) => {
+app.delete('/slambook/:id', async (req, res) => {
   const entryId = req.params.id;
 
   try {

```

Comment:
There should be a return statement after sending the response in order to exit the function